packages/tocco-ui/src/StatedValue/StyledStatedValue.js

```diff
@@ -0,0 +1,130 @@
+import styled, {css} from 'styled-components'
+import _get from 'lodash/get'
+
+import {
+  declareFocus,
+  declareFont,
+  fontScale,
+  shadeColor,
+  spaceScale
+} from '../utilStyles'
+
+/*
+TODO
```

Comment:
i also prefer the polifilly over the react componnent. But shouldn't we load the polyfill with webpack.

Reply:
https://github.com/jonathantneal/focus-within
it this a "real" polyfill? Its important is only overwrites browser behavior if the the browser not supports the feature.

Reply:
see 2e35cdf0404838189dd0b804937b6ede184ed8e4